pennylane/templates/layers/particle_conserving_u1.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Contains the hardware efficient ``ParticleConservingU1`` template.
```

Comment:
I added the U1 since it may be helpful for the user to identify quickly the entangler in the paper. 
@mariaschuld , @soranjh , thoughts on this suggestion by Juan Miguel?

Reply:
Both look fine to me.